joda-money/src/main/java/com/fasterxml/jackson/datatype/jodamoney/AmountRepresenter.java

```diff
@@ -0,0 +1,13 @@
+package com.fasterxml.jackson.datatype.jodamoney;
+
+import org.joda.money.CurrencyUnit;
+import org.joda.money.Money;
+
+import java.math.BigDecimal;
+
+public interface AmountRepresenter<T> {
+
+    T write(Money money);
```

Comment:
Either name is sub-optimal (nothing gets written), or (more likely I think) this should do actual writing via `JsonGenerator` directly. If former, I think this is something like `AmountConverter`, and methods could be something like `fromMoney()` and `toMoney()`.